algorithms/gr-bfsrh/javascript/bfs.js

```diff
@@ -0,0 +1,47 @@
+const bfs = (graph, root, targetNode) => {
+
+  buildPath = (parents, targetNode) => {
+    let result = [targetNode];
+    while (parents[targetNode] !== null) {
+      targetNode = parents[targetNode];
+      result.push(targetNode);
+    }
+    return result.reverse();
+  };
+
+  let parents = [];
+  let queue = [];
+  let visited = [];
+  let current;
+
+  queue.push(root);
+  parents[root] = null;
+  visited[root] = true; // mark first node as visited
+
+  while(queue.length !== -1) {
+    current = queue.shift();
+
+    if (current === targetNode) {
+      return buildPath(parents, targetNode);
```

Comment:
Codacy found an issue: ['buildPath' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6319059)

Reply:
I think the issue here is you have not defined "buildPath" type in line-3. :+1: